itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestHiveDatabaseMetaData.java

```diff
@@ -18,18 +18,15 @@
 
 package org.apache.hive.jdbc;
 
-import org.apache.hive.jdbc.HiveConnection;
-import org.apache.hive.jdbc.Utils;
+import com.google.common.collect.ImmutableMap;
 import org.apache.hive.jdbc.Utils.JdbcConnectionParams;
+import org.junit.Before;
+import org.junit.Test;
 
+import java.sql.SQLException;
+import java.util.HashMap;
 import java.util.LinkedHashMap;
-import java.util.Properties;
 import java.util.Map;
-import java.util.HashMap;
-import java.sql.SQLException;
-
-import org.junit.Before;
-import org.junit.Test;
```

Comment:
Can you just avoid changing the import order, just makes things complicated while backporting

Reply:
Done.